src/Compilers/CSharp/Portable/Compilation/CSharpCompilerDiagnosticAnalyzer.cs

```diff
@@ -36,7 +36,15 @@ internal override ImmutableArray<int> GetSupportedErrorCodes()
                         // We don't support configuring WRN_ALinkWarn. See comments in method "CSharpDiagnosticFilter.Filter" for more details.
                         continue;
 
+                    case (int)ErrorCode.WRN_UnreferencedField:
```

Comment:
@bbarry this is nice except common message provider needs to know about compiler analyzer when two quite two different thing.